updates/julian_1.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO datenbank_mapping(Formular, Datenfeld, de_Beschriftung, Feldtyp, Auswahlherkunft, Array, Seite, gb_beschriftung, fr_beschriftung, la_beschriftung)  VALUES ("freie_suche", "ProvenanceEinzelbeleg", "Provenienz (Einzelbeleg)", "select","selektion_provenance", 0, "freie_suche", "Provenance (Einzelbeleg)", "Provenance (Einzelbeleg)", "Provenance (Einzelbeleg)");
+INSERT INTO datenbank_mapping(Formular, Datenfeld, de_Beschriftung, Feldtyp, Array, Seite, gb_beschriftung, fr_beschriftung, la_beschriftung)  VALUES ("freie_suche", "Ausgabe_Provenance_Einzelbeleg", "Provenienz (Einzelbeleg)", "checkbox", 0, "freie_suche", "Provenance (Einzelbeleg)", "Provenance (Einzelbeleg)", "Provenance (Einzelbeleg)");
+
+INSERT INTO datenbank_mapping(Formular, Datenfeld, de_Beschriftung, Feldtyp, Auswahlherkunft, Array, Seite, gb_beschriftung, fr_beschriftung, la_beschriftung)  VALUES ("freie_suche", "ProvenanceLemma", "Provenienz (Lemma)", "select", "selektion_provenance", 0, "freie_suche", "Provenance (Lemma)", "Provenance (Lemma)", "Provenance (Lemma)");
```

Comment:
Brauchen wir ProvenanceLemma wirklich in der Suche?

Reply:
Ich habe es aufgenommen da es ja Teil des Imports ist und es eine Übersicht liefert der importierten Lemmas.

Aber ich kann es auch wieder entfernen. Was meinst du ?

Reply:
Ja bitte lieber rausnehmen

Reply:
ich habe es entfernt